test-runner-frontend/src/Components/IdCell.js

```diff
@@ -29,16 +30,32 @@ export default function IdCell({ value, hasCheckbox, onChange, status }) {
     <>
       {hasCheckbox ? (
         <Stack direction="row" spacing={2}>
-          <Checkbox value={value} onChange={() => onChange(value)} />
+          {status === 'FINISHED' ? (
+            <Checkbox data-testid="enabledCheckbox" value={value} onChange={() => onChange(value)} />
+          ) : (
+            <Checkbox data-testid="disabledCheckbox" disabled={true} value={value} onChange={() => onChange(value)} />
+          )}
```

Comment:
we don't necessarily need two checkboxes here. we can set the `disabled` attribute to `true` if `status !== 'FINISHED'`

Reply:
Oh nice! I changed it.